bootstrap.sh

```diff
@@ -548,34 +408,36 @@ fi
 STDIN_FILE_DESCRIPTOR="0"
 [ -t "$STDIN_FILE_DESCRIPTOR" ] && STRAP_INTERACTIVE="1"
 
-# We want to always prompt for sudo password at least once rather than doing
-# root stuff unexpectedly.
-sudo_refresh
-
-# Before starting, get the user's code location root where we will clone sentry repos to
-get_code_root_path
+# Prevent sleeping during script execution, as long as the machine is on AC power
+caffeinate -s -w $$ &
 
-install_sentry_cli
+if [ -z "$CODE_ROOT" ]; then
+    read -rp "--> Enter absolute path where we'll clone source code [$HOME/code]: " CODE_ROOT
+else
+    echo "Installing into $CODE_ROOT"
+fi
 
-[ -z "$CI" ] && [ -z "$QUICK" ] && check_github_access
+if [ -z "$CODE_ROOT" ]; then
+    CODE_ROOT="$HOME/code"
+fi
 
-[ "$USER" = "root" ] && abort "Run as yourself, not root."
-groups | grep $Q -E "\b(admin)\b" || abort "Add $USER to the admin group."
+[ -d "$CODE_ROOT" ] || mkdir -p "$CODE_ROOT"
 
-# Prevent sleeping during script execution, as long as the machine is on AC power
-caffeinate -s -w $$ &
+touch "${HOME}/.zshrc"
 
+sudo_refresh
+[ -z "$CI" ] && sudo networksetup -setv6off "Wi-Fi"
```

Comment:
heh is this something we actually want to ship?

Reply:
This feels like it would be better in the troubleshooting CLI tool than here.